doc/api/esm.md

```diff
@@ -1303,7 +1303,7 @@ the CommonJS loader. One of the behavior differences is automatic resolution
 of file extensions and the ability to import directories that have an index
 file.
 
-The `--experimental-specifier-resolution=[mode]` flag can be used to customize
+The `--experimental-specifier-resolution=[resolution-mode]` flag can be used to customize
```

Comment:
This line now exceeds 80 characters, while the rest of the paragraph is wrapped at 80 chars. Not sure whether we usually fix this.

Reply:
Yes, it fails the lint doc check, which blocks merging.

Reply:
@tniessen in my anecdotal experience we usually fix this for first time contributors if they are unable to do so themselves.